files/es/web/html/element/input/number/index.md

```diff
@@ -5,7 +5,7 @@ translation_of: Web/HTML/Element/input/number
 browser-compat: html.elements.input.type_number
 ---
 
-{{HTMLRef("Input_types")}}
+{{HTMLSidebar("Input_types")}}
```

Comment:
I checked the [English version](https://github.com/mdn/content/edit/main/files/en-us/web/html/element/input/number/index.md) and it didn't include the argument, so, I suggest removing it to match English file. 

```suggestion
{{HTMLSidebar)}}
```

Reply:
```suggestion

{{HTMLSidebar}}

```